File: UniPharApi/Models/ServiceModel.cs
namespace UniPharApi.Models;

public class ServiceModel
{
    public string Id { get; set; }
    public string Title { get; set; }
    public string Slug { get; set; }
    public string? HeroHeading { get; set; }
    public string? HeroSubtext { get; set; }
    public string? HeroImageUrl { get; set; }
    public string? Description { get; set; }
    public string? IconUrl { get; set; }
    public bool IsFeatured { get; set; }
    public List<FeatureModel> Features { get; set; } = new();
}

File: uniphar-frontend/src/app/models/service.model.ts
import { BreadcrumbItem } from "./page.model";

export interface Service {
  id: string;
  title: string;
  slug: string;
  heroHeading: string;
  heroSubtext: string;
  heroImageUrl: string | null;
  description: string;
  iconUrl: string | null;
  isFeatured: boolean;
  features: Feature[];
  breadcrumbs: BreadcrumbItem[];
}

export interface Feature {
  title: string;
  description: string;
  iconUrl: string | null;
  isHighlighted: boolean;
}